v3/src/components/graph/adornments/movable-value/movable-value-adornment-handler.ts

```diff
@@ -1,11 +1,65 @@
 import { DIAdornmentHandler } from "../../../../data-interactive/handlers/adornment-handler"
+import { adornmentNotFoundResult, adornmentNotSupportedByPlotTypeResult,
+  invalidValuesProvidedeResult, valuesRequiredResult } from "../../../../data-interactive/handlers/di-results"
 import { IGraphContentModel } from "../../models/graph-content-model"
-import { IAdornmentModel } from "../adornment-models"
-import { isMovableValueAdornment } from "./movable-value-adornment-model"
-import { AdornmentData, adornmentMismatchResult, cellKeyToCategories } from "../utilities/adornment-handler-utils"
+import { getAdornmentContentInfo } from "../adornment-content-info"
+import { IAdornmentModel, IUpdateCategoriesOptions } from "../adornment-models"
+import { IAdornmentsBaseStore } from "../store/adornments-base-store"
+import { AdornmentData, adornmentMismatchResult, cellKeyToCategories, isAdornmentSupportedByPlotType }
+  from "../utilities/adornment-handler-utils"
+import { IMovableValueAdornmentModel, isMovableValueAdornment } from "./movable-value-adornment-model"
 import { kMovableValueType } from "./movable-value-adornment-types"
 
 export const movableValueAdornmentHandler: DIAdornmentHandler = {
+  create(args) {
+    const { graphContent } = args
+    const isAdornmentSupported = isAdornmentSupportedByPlotType(kMovableValueType, graphContent.plotType)
+    if (!isAdornmentSupported) return adornmentNotSupportedByPlotTypeResult
+
+    const adornmentsStore = graphContent.adornmentsStore as IAdornmentsBaseStore
+    const dataConfig = graphContent.dataConfiguration
+    const cellKeys = dataConfig?.getAllCellKeys()
+    const data: AdornmentData[] = []
+    const existingMovableValueAdornment =
+      adornmentsStore.findAdornmentOfType<IMovableValueAdornmentModel>(kMovableValueType)
+    const componentContentInfo = getAdornmentContentInfo(kMovableValueType)
+    const adornment =
+      existingMovableValueAdornment ?? componentContentInfo.modelClass.create() as IMovableValueAdornmentModel
+
+    if (!existingMovableValueAdornment) {
+      const options: IUpdateCategoriesOptions = { ...graphContent.getUpdateCategoriesOptions(), addMovableValue: true }
+      adornmentsStore.addAdornment(adornment, options)
```

Comment:
Shouldn't `create` create an additional movable value, even if there's already one present? And shouldn't the API allow the client to specify the value rather than using a default value?